app/templates/agenda-items/index.hbs

```diff
@@ -78,12 +78,13 @@
           {{/if}}
         {{/let}}
       </div>
-      <FilterSidebar 
-        @id="filtersidebar" 
+      <FilterSidebar
+        @id="filtersidebar"
         @class="c-interface__sidebar c-interface__sidebar--dialog {{if this.hasFilter 'is-visible'}}"
         {{focus-trap
           isActive=this.hasFilter
           shouldSelfFocus=true
+          additionalElements=(array "#ember-basic-dropdown-wormhole")
```

Comment:
The range picker doesn't work properly on mobile either, but that's being replaced by #88 so I didn't include that container here.